mm2src/coins/utxo/slp.rs

```diff
@@ -72,9 +151,30 @@ impl MarketCoinOps for SlpToken {
                     .await?;
                 let prev_tx: UtxoTx = deserialize(prev_tx_bytes.0.as_slice()).unwrap();
                 let script: Script = prev_tx.outputs[0].script_pubkey.clone().into();
+                if let Ok(slp_data) = parse_slp_script(&script) {
+                    match slp_data.transaction {
+                        SlpTransaction::Send { token_id, amount } => {
+                            if H256::from(token_id.as_slice()) == coin.token_id && amount.len() == 8 {
+                                let satoshi = u64::from_be_bytes(amount.try_into().unwrap());
+                                let decimal = big_decimal_from_sat_unsigned(satoshi, coin.decimals);
+                                spendable += decimal;
+                            }
+                        },
+                        SlpTransaction::Genesis {
+                            initial_token_mint_quantity,
+                            ..
+                        } => {
+                            if prev_tx.hash().reversed() == coin.token_id && initial_token_mint_quantity.len() == 8 {
+                                let satoshi = u64::from_be_bytes(initial_token_mint_quantity.try_into().unwrap());
```

Comment:
How about collecting the `SlpTransaction::Genesis::initial_token_mint_quantity` as [u8; 8] instead of `Vec<u8>`? Is it possible?

Reply:
Might be, I will check how I can do it with the `parity-bitcoin::serialization` crate.